largest_prime_factor.py

```diff
@@ -0,0 +1,27 @@
+"""
+The prime factors of 13195 are 5, 7, 13 and 29.
+
+What is the largest prime factor of the number 600851475143? 
+"""
+def is_prime(n):
```

Comment:
nice abstraction to a helper function! would be good to add some unit tests for this function as well ;)

Reply:
thank you :D